### PR TITLE
v1.1: events cleanup

### PR DESCRIPTION
**Pull request by mPaella**
seperating internal vs public events, better organization of event names vs payloads

### CODE REVIEW

File: packages/core/base/src/types/events/public/events.ts [hunk 1]
```diff
@@ -0,0 +1,59 @@
+// Payment events
```
Comment (mPaella) [inline packages/core/base/src/types/events/public/events.ts:1]: nothing changed here, just organized

File: packages/core/base/src/types/events/public/payloads.ts [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+    [CrossmintEvents.PAYMENT_PREPARATION_FAILED]: CrossmintEventErrorPayload;
+    [CrossmintEvents.PAYMENT_PROCESS_SUCCEEDED]: PaymentCompletedPayload;
+    [CrossmintEvents.PAYMENT_PROCESS_CANCELED]: EmptyObject;
+    [CrossmintEvents.PAYMENT_PROCESS_REJECTED]: PaymentRejectedPayload;
+    [CrossmintEvents.ORDER_PROCESS_STARTED]: EmptyObject;
+    [CrossmintEvents.TRANSACTION_FULFILLMENT_SUCCEEDED]: TransactionFulfillmentSucceededPayload;
+    [CrossmintEvents.TRANSACTION_FULFILLMENT_FAILED]: TransactionFulfillmentFailedPayload;
+    [CrossmintEvents.ORDER_PROCESS_FINISHED]: OrderProcessFinishedPayload;
+    [CrossmintEvents.RECIPIENT_WALLET_CHANGED]: RecipientWalletChangedPayload;
+    [CrossmintEvents.RECIPIENT_EMAIL_CHANGED]: RecipientEmailChangedPayload;
```
Comment (guillermodoghel) [inline packages/core/base/src/types/events/public/payloads.ts:20]: Can you add one for the serialized tx?

{ serializedTx, currency maybe? }

File: packages/core/base/src/types/events/internal/events.ts [hunk 3]
```diff
@@ -0,0 +1,8 @@
+// All internal events
+export const CrossmintInternalEvents = {
+    PARAMS_UPDATE: "params-update",
+    UI_HEIGHT_CHANGED: "ui:height.changed",
+    CRYPTO_PAYMENT_USER_ACCEPTED: "crypto-payment:user-accepted",
+    CRYPTO_PAYMENT_USER_REJECTED: "crypto-payment:user-rejected",
```
Comment (guillermodoghel) [inline packages/core/base/src/types/events/internal/events.ts:6]: maybe add crypto-payment:serialized-transaction

**Review by guillermodoghel — APPROVED** [review]
lgtm

**Pull request merged**